### ISSUE TITLE
Data set 

### ISSUE THREAD

**Issue by yuan-cherish** [attilanagy234/neural-punctuator#14]
Hello, can you share the data set of the experiment, the displayed network link can no longer be downloaded.Thanks!

**Comment by attilanagy234** [comment]
Hey there! Which dataset are you trying to access? Both links seems to work fine for me.

**Comment by yuan-cherish** [comment]
Thanks very much for your reply!Because I am currently studying the direction of automatic punctuation repair, I need some TED corpus and training set, development set and test set. If you can, I hope you can give me the following data, thank you very much!

http://hltshare.fbk.eu/IWSLT2012/IWSLT12.TALK.train.en http://hltc.cs.ust.hk/iwslt/index.php/evaluation-campaign/ted-task.html#TrainingData

http://hltshare.fbk.eu/IWSLT2012/IWSLT12.SLT.dev2010.tgz

http://hltshare.fbk.eu/IWSLT2012/IWSLT12.SLT.dev2010.1best.v1.tgz

http://hltshare.fbk.eu/IWSLT2012/IWSLT12.SLT.tst2010.tgz

http://hltshare.fbk.eu/IWSLT2012/IWSLT12.SLT.tst2010.1best.v1.tgz

http://hltshare.fbk.eu/IWSLT2012/IWSLT12.TALK.ASRtestsets.tgz





------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "attilanagy234/neural-punctuator"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年8月24日(星期二) 晚上7:58
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [attilanagy234/neural-punctuator] Data set  (#14)





 
Hey there! Which dataset are you trying to access? Both links seems to work fine for me.
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by attilanagy234** [comment]
You should be able to download the datasets from here:
- Train & dev: https://wit3.fbk.eu/2012-03
- Test: https://wit3.fbk.eu/2012-03-b
